src/backend/distributed/citus--8.3-1--8.4-1.sql

```diff
@@ -21,6 +83,15 @@ BEGIN
     -- enterprise catalog tables
     CREATE TABLE public.pg_dist_authinfo AS SELECT * FROM pg_catalog.pg_dist_authinfo;
     CREATE TABLE public.pg_dist_poolinfo AS SELECT * FROM pg_catalog.pg_dist_poolinfo;
+
+    -- store upgrade stable identifiers on pg_dist_object catalog
```

Comment:
Is this to prevent anyone else to mess with the tables in non-upgrade times?  Might be good to add a comment since it is unusual and not trivial to understand.

Reply:
yeah, it is strange, and as discussed verbally this will change to a situation where the columns are always there.